backend/src/schema/resolvers/posts.js

```diff
@@ -211,10 +225,68 @@ export default {
       })
       return emoted
     },
+    pinPost: async (_parent, params, context, _resolveInfo) => {
+      let pinnedPost
+      const { driver, user } = context
+      const session = driver.session()
+      const { id: userId } = user
+      let writeTxResultPromise = session.writeTransaction(async transaction => {
+        const deletePreviousRelationsResponse = await transaction.run(
+          `
+          MATCH ()-[previousRelations:PINNED]->(post:Post)
+          DELETE previousRelations
```

Comment:
What happens with the posts are not pinned anymore?
They appear as usual post?
That would be annoying, or?

Reply:
Yes, what should happen with them? They are part of the network... Why should they be disabled or deleted? Or do you mean something else

Reply:
And how long does it take before an admin can unpin a post? I was thinking of a case that one admin has just pinned a post, and immediately after another admin pins a different post.

Reply:
You can pin another `Post` straight after @aonomike... We shouldn't have an issue like you described cause we have very few admins that will pin posts, and hopefully will not abuse pinning them.
If we do have an issue, we can address it.